Util/Xamarin.Build.Download/source/Xamarin.Build.Download/Xamarin.Build.Download.targets

```diff
@@ -107,9 +107,23 @@
 		</ItemGroup>
 	</Target>
 
+	<!-- TODO: AndroidAarLibrary is not yet in AvailableItemName so we added it manually for now -->
+	<ItemGroup>
+		<AvailableItemName Include="AndroidAarLibrary" />
+	</ItemGroup>
+	<!-- END TODO -->
+		
+	<Target Name="_XamarinBuildDownloadAarInclude"
+			DependsOnTargets="_XamarinBuildDownload"
+			Condition="'@(XamarinBuildDownloadRestoreAssemblyAar)'!='' and %(AvailableItemName.Identity) == 'AndroidAarLibrary'">
```

Comment:
Thinking about it , it might be better to define a property 

```xml
<PropertyGroup>
   <SupportsAarLibrary Condition="'%(AvailableItemName.Identity)' == 'AndroidAarLibrary'">true</SupportsAarLibrary>
<SupportsAarLibrary Condition="'$(SupportsAarLibrary)' == ''">false</SupportsAarLibrary>
</PropertyGroup>
```

I know MSbuild might run a target multiple times when using `%(foo.Identity)` due to its Batching support. I'm not sure if that happens in this case. It might not, if it doesn't ignore me :)